meeting-notes/notes-January-04-2021.md

```diff
@@ -0,0 +1,23 @@
+
+## Week of January 04, 2021
+
+### What can we talk about?
+
+- [TODO]
```

Comment:
I could use some help thinking about how we structure the billing project(s), and how we assign DRIs to them: 

- Entitlements changes (April)
- Container Registry billing (TBD)
- GHAE Billing (April)